scripts/completion/dvc.bash

```diff
@@ -8,35 +8,44 @@
 #   - https://opensource.com/article/18/3/creating-bash-completion-script
 #----------------------------------------------------------
 
-_dvc_commands='init destroy add import checkout run pull push fetch \
-              status repro remove move unprotect gc config remote metrics \
-              install root lock unlock pipeline commit'
+_dvc_commands='init get get_url destroy add remove move unprotect run \
+              repro pull push fetch status gc import import_url config \
+              checkout remote cache metrics install root lock unlock \
+              pipeline commit diff version'
 
-_dvc_options="-h --help -v --version"
-_dvc_global_options="-h --help -q --quiet -V --verbose"
+_dvc_options="-h --help -V --version"
+_dvc_global_options="-h --help -q --quiet -v --verbose"
 
-_dvc_add="-R --recursive -f --file --no-commit"
-_dvc_checkout="$(compgen -G '*.dvc')"
+_dvc_add="-R --recursive -f --file --no-commit $(compgen -G '*')"
+_dvc_cache=""
+_dvc_checkout="-d --with-deps -R --recursive -f --force $(compgen -G '*.dvc')"
+_dvc_commit="-d --with-deps -R --recursive -f --force $(compgen -G '*.dvc')"
 _dvc_config="-u --unset --local --system --global"
 _dvc_destroy="-f --force"
-_dvc_fetch="--show-checksums -j --jobs -r --remote -a --all-branches -T --all-tags -d --with-deps -R --recursive"
-_dvc_gc="-a --all-branches -T --all-tags -c --cloud -r --remote -f --force -p --project"
-_dvc_import="--resume"
+_dvc_diff="-t --target"
+_dvc_fetch="--show-checksums -j --jobs -r --remote -a --all-branches -T --all-tags -d --with-deps -R --recursive $(compgen -G '*.dvc')"
+_dvc_gc="-a --all-branches -T --all-tags -c --cloud -r --remote -f --force -p --projects -j --jobs"
+_dvc_get="-o --out --rev"
+_dvc_get-url=""
```

Comment:
@mroutis are you sure `-` syntax is correct? In my zsh this doesn't work.

Reply:
my bad! The syntax is incorrect, indeed, but having `_dvc_get_url` will result in completing `get_url` instead of `get-url` maybe we would need to do some adjustments to the `COMREPLY` variable, I'll think through this and comment back later :)